src/command/check-secret-expiry.ts

```diff
@@ -6,43 +6,38 @@ import { AWSCredentials, ProfileConfiguration } from '../mfa-login';
 import { getKeyAgeInDays } from '../lib/time';
 import { createIAMClient, createStsClient } from '../lib/create-aws-client';
 
-const getExpiryStatus = (
-  dateCreated: Date,
-  maxAge: number | undefined
+const getKeyAgeStatus = (
+  keyAge: number,
+  keyMaxAge: number
 ): { text: string; textColor: chalk.Chalk } | undefined => {
-  let textColor;
-  const keyAge = getKeyAgeInDays(dateCreated.getTime());
-
-  if (maxAge) {
-    const daysLeft = maxAge - keyAge;
-
-    if (daysLeft < 0) {
-      return { text: `✘ has expired`, textColor: chalk.red };
-    } else if ([0, 1].includes(daysLeft)) {
-      return {
-        text: `${daysLeft === 1 ? '⚠ expires tomorrow' : '⚠ expires today'}`,
-        textColor: chalk.yellow,
-      };
-    } else if (daysLeft < 7) {
-      return { text: `⚠ expires in ${daysLeft} days`, textColor: chalk.yellow };
-    } else if (daysLeft < 30) {
-      return {
-        text: `✓ expires in ${keyAge} days`,
-        textColor: chalk.green,
-      };
-    } else return;
-  } else {
-    if (keyAge < 10) return;
+  const daysLeft = keyMaxAge - keyAge;
+
+  if (daysLeft < 0) {
+    return { text: `✘ maximum age is exceeded`, textColor: chalk.red };
+  } else if ([0, 1].includes(daysLeft)) {
+    return {
+      text: `${daysLeft === 1 ? '⚠ maximum age exceeds tomorrow' : '⚠ maximum age exceeds today'}`,
+      textColor: chalk.yellow,
+    };
+  } else if (daysLeft < 7) {
+    return { text: `⚠ maximum age exceeds in ${daysLeft} days`, textColor: chalk.yellow };
```

Comment:
exceeds --> `will be reached`

Reply:
Don't resolve things if you don't make changes or leave a comment please

Reply:
I am a bit confused.
![image](https://user-images.githubusercontent.com/61769420/147685905-3af32c76-633e-4db3-9a55-9cc80ee7a0b7.png)

I think your comment was referring to line 19 and 23 originally, and I changed the text accordingly. 
Thats why it was resolved.